src/lib/prismaUtils.ts

```diff
@@ -460,3 +460,40 @@ export function findStakedDeposits({
       throw new Error(`Unknown namespace: ${namespace}`);
   }
 }
+
+export function getProposalsCount({
+  namespace,
+  contract,
+}: {
+  namespace: TenantNamespace;
+  contract: string;
+}) {
+  const condition = {
+    where: {
+      contract,
+    },
+  };
+
+  switch (namespace) {
+    case TENANT_NAMESPACES.OPTIMISM:
+      return prisma.optimismProposals.count(condition);
```

Comment:
And, regardless of if you keep the prisma based approach or switch to SQL, we need to only count proposals that are not cancelled and not snapshots, if we want to generalize this reliably.  Not mandatory here, but in the case of a new tenant, they could easily make 3 or 4, and then cancel all of them.

Reply:
This is great feedback @jefag, I am going to merge but fix in another PR. Ty